docs/running-on-mesos.md

```diff
@@ -753,6 +753,16 @@ See the [configuration page](configuration.html) for information on Spark config
     <code>spark.cores.max</code> is reached
   </td>
 </tr>
++<tr>
```

Comment:
this has an extra `+<tr>`

also if you are at it could you revert the other whitespace changes above?

Reply:
+1 for reverting whitespaces. Let's leave this minimised and targeted.

Reply:
Ok sure.